packages/code-du-travail-data/search/indexing/document_slug_checker.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+import sys
+
+from search.indexing.cdtn_documents import CDTN_DOCUMENTS
+
+
+console = logging.StreamHandler()
+logger = logging.getLogger(__name__)
+logger.addHandler(console)
+logger.setLevel(logging.INFO)
+
+
+# documents_slug_checker will return a list of dict of duplicate slugs from documents
+# thanks to @adipasquale
+# CDTN_DOCUMENTS = [
+#   {'slug': 'blah1', 'source': 'truc'},
+#   {'slug': 'blah1', 'source': 'truc'},
+#   {'slug': 'blah2', 'source': 'truc'},
+#   {'slug': 'blah1', 'source': 'truc'}
+# ]
+
+def documents_slug_checker():
+    slugs = [f"{item.get('source')}/{item.get('slug')}" for item in CDTN_DOCUMENTS]
```

Comment:
again, I think I think you should rather use `item['source']` and `item['slug']` so that it throws an exception when an item doesn't have one of these keys